### DISCUSSION TITLE
Hair meshes visibility option

### DISCUSSION THREAD

**Discussion by mokujinh** [Mustard2/MustardUI#34] [Ideas]
Adding a shortcut "hide for render" option for hair meshes could be really interesting if you plan to add helmets and/or hats for models as extra outfits. The current version of the script has an option to hide hair meshes and armatures but only for the viewport.

**Comment by Mustard2** [comment]
Sorry I missed this, the discussions don't send me any notification. :(
You can open an issue next time, it will be faster. :)

**Comment by Mustard2** [comment]
I opened an issue to keep track of this: https://github.com/Mustard2/MustardUI/issues/38
